.circleci/config.yml

```diff
@@ -12,9 +12,11 @@ references:
       - *npm_cache_key
   # Note: important that values of guids and keys have matching order!"
   study_keys: &study_keys
-    "osteo brain angio mbc testboston mpc prion atcp rarex rgp esc circadia pancan"
+    "osteo brain angio mbc testboston mpc prion atcp rarex rgp esc cgc circadia pancan"
   study_guids: &study_guids
-    "CMI-OSTEO cmi-brain ANGIO cmi-mbc testboston cmi-mpc PRION atcp rarex RGP cmi-esc circadia cmi-pancan"
+    "CMI-OSTEO cmi-brain ANGIO cmi-mbc testboston cmi-mpc PRION atcp rarex RGP cmi-esc cgc circadia cmi-pancan"
+  dispatch_secret_id: &dispatch_secret_id
+      app-engine-dispatch
```

Comment:
Looks like a bad merge. `dispatch_secret_id` should have been removed.

Reply:
Fixed this issue